homeassistant/components/deluge/__init__.py

```diff
@@ -42,7 +42,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         await hass.async_add_executor_job(api.connect)
     except (ConnectionRefusedError, TimeoutError, SSLError) as ex:
         raise ConfigEntryNotReady("Connection to Deluge Daemon failed") from ex
-    except Exception as ex:  # noqa: BLE001
+    except Exception as ex:
```

Comment:
Why is the noqa not needed here?

Reply:
Because of the raise from inside

Reply:
But it's not raised in all cases.

Reply:
Yes, that is currently not taken into account. I can revert this and fix it in Ruff.

Reply:
That sounds good. 👍